tmp/lanelet2_extension/include/lanelet2_extension/utility/utilities.hpp

```diff
@@ -111,7 +111,10 @@ bool isInLanelet(
   const double radius = 0.0);
 geometry_msgs::msg::Pose getClosestCenterPose(
   const lanelet::ConstLanelet & lanelet, const geometry_msgs::msg::Point & search_point);
-
+double getLateralDistanceToLanelet(
```

Comment:
Maybe changing the function name to `getLateralDistanceToLaneletCenterline` or `getLateralDistanceToCenterline` might be better. Otherwise we don't know if it's a distance to the lane bound or to the centerline

Reply:
Sounds great

Reply:
done